source/train/DataSystem.py

```diff
@@ -261,20 +294,21 @@ def print_summary(self,
         # width 65
         sys_width = 42
         tmp_msg += "---Summary of DataSystem------------------------------------------------\n"
-        tmp_msg += "find %d system(s):\n" % self.nsystems
+        tmp_msg += "found %d system(s):\n" % self.nsystems
         tmp_msg += "%s  " % self._format_name_length('system', sys_width)
-        tmp_msg += "%s  %s  %s  %5s\n" % ('natoms', 'bch_sz', 'n_bch', 'prob')
+        tmp_msg += "%s  %s  %s   %s  %5s\n" % ('natoms', 'bch_sz', 'n_bch', "n_test", 'prob')
```

Comment:
that is not necessary, `n_test` spans 6 characters so it is pointless to apply additional formatting to `%6s` as it will not change anything in the output. `prob` is the last word so it is also not useful to apply `%6s` formatting as it will only add two spaces before the end of the line.